data/sql/updates/pending_db_world/wg_banners_fix.sql

```diff
@@ -0,0 +1,34 @@
+-- Fix invalid banner name
+UPDATE `gameobject_template` SET `name`='Alliance Banner' WHERE (`entry`=192269);
+
+-- Rotate existing banners
+UPDATE `gameobject` SET `orientation`=4.71, `rotation2`=0, `rotation3`=0 WHERE (`guid`=75186);
+UPDATE `gameobject` SET `orientation`=4.71, `rotation2`=0, `rotation3`=0 WHERE (`guid`=75190);
+UPDATE `gameobject` SET `orientation`=4.71, `rotation2`=0, `rotation3`=0 WHERE (`guid`=75191);
+UPDATE `gameobject` SET `orientation`=4.71, `rotation2`=0, `rotation3`=0 WHERE (`guid`=75179);
+
+-- Add missing Fortress banners
```

Comment:
to avoid any issues with existing data, an insert should be preceded by a "delete if exist..."